### PR TITLE
refactor: deck

### PR DESCRIPTION
**Pull request by JuanPablo2655**
`deck.rs` is now a proper deck of cards

### CODE REVIEW

**Comment by JuanPablo2655** [comment]
After playtesting for a bit, I realized that the deck isn't shuffled lol.

**Review by rand0m-cloud — APPROVED** [review]
LGTM! In a follow-up PR, work on a Deck::new_standard_deck and a Deck::shuffle.

File: src/lib/deck.rs [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
+        for value in 1..=13 {
+            for suit in Suit::all() {
+                cards.push_back(Card::new(value, suit))
+            }
+        }
+        Self { cards }
+    }
+
+    fn from_cards(cards: &[Card]) -> Self {
+        let cards = VecDeque::from(cards.to_vec());
```
Comment (rand0m-cloud) [inline src/lib/deck.rs]: Add `cargo clippy` to your build process. I believe `VecDeque::from(&cards)` is sufficient

**Review by rand0m-cloud — COMMENTED** [review]
Run `cargo clippy` and fix all the lints. Otherwise, it LGTM!

**Review by rand0m-cloud — APPROVED** [review]

**Pull request merged**